script.js

```diff
@@ -1,4 +1,53 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+
+var scissors  = "scissors"
+var stone = "stone"
+var paper = "paper"
```

Comment:
Well done Joseph for thinking this ahead

There is something know as [SCREAM_CASE constants ](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/) which is what you did, but instead of small letters, using capital letters would be a way to show these variables do not change.

It's simply another way to name variables that remain unchanged through out the program eg.
```
var STONE = "stone"
var SCISSORS = "scissors"
var PAPER = "paper"
```